pandas/tests/plotting/frame/test_frame.py

```diff
@@ -787,6 +787,18 @@ def test_plot_scatter_with_s(self):
         ax = df.plot.scatter(x="a", y="b", s="c")
         tm.assert_numpy_array_equal(df["c"].values, right=ax.collections[0].get_sizes())
 
+    def test_plot_scatter_with_norm(self):
+        # added while fixing GH 45809
+        import matplotlib as mpl
+
+        ax = self.hexbin_df.plot.scatter(x="A", y="B", c="C", norm=mpl.colors.LogNorm())
+        assert ax.collections[0].norm._scale
```

Comment:
Yeah comparing "private" attributes will probably be brittle in the future. Is it possible to just compare `assert ax.collections[0].norm is mpl.colors.LogNorm()` or something similar?

Reply:
I'm fairly sure I tried both an `==` and `is`, and neither worked. The value of `ax.collections[0].norm` seems to be the same regardless of whether `norm` is set or not. I'll double check, hopefully I'll get time this week.